README.md

```diff
@@ -1 +1,70 @@
-# yesterdays_news_admin
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
You can set a title in the readme file like: "Yesterdays News (Admin)"

Reply:
Roger!